internal/graph/model/id.go

```diff
@@ -19,7 +19,7 @@ import (
 	"io"
 	"strings"
 
-	"github.com/kjk/smaz"
+	"github.com/epk/smaz"
```

Comment:
https://github.com/kjk/smaz does not have a `go.mod` and has a relative import which is no longer allowed  in Go 1.18

Diff here: https://github.com/epk/smaz/commit/c11a89997fcfe65ded3d3bc79694ce50e675f766

Reply:
https://github.com/upbound/xgql/blob/main/internal/graph/model/id.go#L115

Note that updates to smaz logic could result in a breaking change for xgql per the comment above. That said, at a glance this change seems safe to me.

Reply:
Yeah, I only added a `go.mod`, removed relative imports and ran the code through https://github.com/mvdan/gofumpt